CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- Please document PR changes here. -->
 
+### Changed
+
+* `expandToSnippet` of element typeahead autocompletions now includes slots for autocompletion.
```

Comment:
Probably easier to understand if you call them `children corresponding to shadow dom slots`